app/views/courses/_about_schools.html.erb

```diff
@@ -1,5 +1,13 @@
 <div class="govuk-!-margin-bottom-8">
   <h2 class="govuk-heading-l" id="section-schools"><%= course.placements_heading %></h2>
+  <aside class="app-advice" role="complementary">
+    <p class="app-advice__caption">Advice from Get into teaching</p>
+    <h3 class="app-advice__title">Where you will train</h3>
+    <div class="app-advice__body">
+      <p class="govuk-body">This course has placement schools. Most of your time will be spent in the classroom with experienced teachers.</p>
```

Comment:
Travel to work information is not yet available. This means we cannot add the travel area to the 'where you will train' Get into Teaching Aside. We'll need a ticket to add this in when ready.